dhis2-android-app/src/main/java/org/dhis2/mobile/ui/activities/DataEntryActivity.java

```diff
@@ -92,7 +92,7 @@ public static void navigateTo(Activity activity, DatasetInfoHolder info) {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_data_entry);
-        setupToolbar();
+        setupToolbar(null);
```

Comment:
we should avoid set parameters to null because is a potential NullPointerException inside the SetupToolBar method. I think better if we create a method ChangeTitle that is invoked when is necessary and SetupToolBar method that not change the title.